### PR TITLE
feat: add sandbox and transfer icons

### PR DESCRIPTION
**Pull request by steue**
- [ ] **BREAKING CHANGE?** 

## Description

Adds the following icons:

- `12/sandbox-stroke.svg`
- `12/sandbox-fill.svg`
- `16/sandbox-stroke.svg`
- `16/sandbox-fill.svg`
- `12/phone-call-transfer-only-stroke.svg`
- `12/phone-call-transfer-only-fill.svg`
- `16/phone-call-transfer-only-stroke.svg`
- `16/phone-call-transfer-only-fill.svg`

## Checklist

* [x] :ok_hand: SVG updates are Garden Designer approved (add the
  designer as a reviewer)
* [x] :globe_with_meridians: SVG demo is up-to-date (`yarn start`)
* [x] :black_medium_small_square: Renders as expected in "dark" mode
* [x] :white_large_square: Renders as expected @ 2x scale

### CODE REVIEW

File: src/12/sandbox-fill.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="12px" height="12px" viewBox="0 0 12 12">
+  <g fill="none">
```
Comment (jzempel) [inline src/12/sandbox-fill.svg:3]: Please remove this unnecessary `g`roup.

File: src/12/sandbox-stroke.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="12px" height="12px" viewBox="0 0 12 12">
+  <g fill="none">
```
Comment (jzempel) [inline src/12/sandbox-stroke.svg:3]: Please remove this unnecessary `g`roup.

File: src/16/sandbox-fill.svg [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="16px" height="16px" viewBox="0 0 16 16">
+  <g fill="none">
```
Comment (jzempel) [inline src/16/sandbox-fill.svg:3]: Please remove this unnecessary `g`roup.

File: src/16/sandbox-stroke.svg [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="16px" height="16px" viewBox="0 0 16 16">
+  <g fill="none">
```
Comment (jzempel) [inline src/16/sandbox-stroke.svg:3]: Please remove this unnecessary `g`roup.

**Review by jzempel — CHANGES_REQUESTED** [review]

File: src/16/sandbox-stroke.svg (hunk 4 above)
Comment (steue) [reply src/16/sandbox-stroke.svg]: ah! great catch. thank you jz! ⚡️ 💪

**Review by jzempel — APPROVED** [review]

**Review by nyckie — APPROVED** [review]

**Comment by jzempel** [comment]
- `12/sandbox-stroke.svg`
- `12/sandbox-fill.svg`
- `16/sandbox-stroke.svg`
- `16/sandbox-fill.svg`
- `12/phone-call-transfer-only-stroke.svg`
- `12/phone-call-transfer-only-fill.svg`
- `16/phone-call-transfer-only-stroke.svg`
- `16/phone-call-transfer-only-fill.svg`

**Pull request merged**